<launch>
    <node pkg='line_follower' type='path_finder.py' name='path_finder' output='screen'></node>
    <param name="line_follower_speed" value="0.2" type="double"/>
</launch>
